### PR TITLE
remove preview from extension

### PR DESCRIPTION
**Pull request by elahehrashedi**
remove the preview from extension
https://github.com/microsoft/vscode-cpptools/issues/6047

### CODE REVIEW

File: Extension/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [![Badge](https://aka.ms/vsls-badge)](https://aka.ms/vsls)
 
-This preview release of the C/C++ extension adds language support for C/C++ to Visual Studio Code, including features such as IntelliSense and debugging.
+This release of the C/C++ extension adds language support for C/C++ to Visual Studio Code, including features such as IntelliSense and debugging.
```
Comment (bobbrow) [inline Extension/README.md]: replace "This release of the" with just "The"

File: Extension/package.json [hunk 2]
```diff
@@ -2,9 +2,8 @@
   "name": "cpptools",
   "displayName": "C/C++",
   "description": "C/C++ IntelliSense, debugging, and code browsing.",
-  "version": "0.30.0-master",
+  "version": "1.0.0",
```
Comment (bobbrow) [inline Extension/package.json]: 1.0.0-master

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 [![Badge](https://aka.ms/vsls-badge)](https://aka.ms/vsls)
 
-This preview release of the C/C++ extension adds language support for C/C++ to Visual Studio Code, including features such as IntelliSense and debugging.
+This release of the C/C++ extension adds language support for C/C++ to Visual Studio Code, including features such as IntelliSense and debugging.
```
Comment (bobbrow) [inline README.md]: Same update as the other readme

**Review by bobbrow — CHANGES_REQUESTED** [review]

**Review by bobbrow — APPROVED** [review]

**Comment by bobbrow** [comment]
Let @sean-mcmanus merge it once he's done building the insiders

File: README.md [hunk 4]
```diff
@@ -52,9 +52,11 @@ The extension has platform-specific binary dependencies, therefore installation
  Package | Platform
 :--- | :---
 `cpptools-linux.vsix` | Linux 64-bit
-`cpptools-linux32.vsix` | Linux 32-bit ([available up to version 0.27.0](https://github.com/microsoft/vscode-cpptools/issues/5346))
+`cpptools-linux-armhf.vsix` | Linux ARM
```
Comment (sean-mcmanus) [inline README.md]: I just realized this might be better as Linux ARM 32-bit.

**Review by sean-mcmanus — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
> Let @sean-mcmanus merge it once he's done building the insiders

0.30.0-insiders5 is released.

**Pull request merged**